deploy/clowdapp.yaml

```diff
@@ -0,0 +1,199 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: approval-process
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: approval-process
```

Comment:
The name here should be `approval`, right? So after combining with deployment name, it will be `approval-pam`

Reply:
We have to avoid `approval` because there is one DB instance created in the same namespace for approval-api already. The deployment name now becomes apprroval-process-pam.